README.md

```diff
@@ -694,8 +694,8 @@ configuration.
 configuration in most cases.)
 
 Any incoming request that has a request body that exceeds this limit will be
-rejected with a `413` (Request Entity Too Large) error message without calling
-the next middleware handlers.
+accepted but their request body will not be added to the request. Browsers consider 
+a request failed when a response is sent before the entire request has gone out.
```

Comment:
This is valid remark, but it is actually unrelated to this PR. I will remove this line for now and will make sure file a new ticket to keep track of this :+1:

Reply:
See  #273 for follow-up discussion about this :+1: